### PR TITLE
E3 Conference Website 

### PR DESCRIPTION
**Pull request by ramiaquista**
- ### Linters:
   Added HTML/CSS and JavaScript linters.
- ### HTML: 
   Added an HTML Home Page (index.html)
   Added an HTML About Page (about-page.html)
- ### CSS:
   Added CSS styling to Home Page and About Page (style.css)
   Added CSS media query to manage and make the entire website responsive for different sizes.
- ### JavaScript:
   Added JS Scripts to make a dynamic HTML about speakers data when the page load's (speakers-data.js)
   Added Js Script functionality to my own mobile navbar (menu-options.js)
- ### Extras:
   Added FontAwesome 5 icons
   Added custom background images on HomePage and AboutPage
   Added customized content about E3 Gaming Event.
   Added Bootstrap to desktop navbar version.
   Added and Designed own mobile navbar
- ### About:
   Added Licenses and Credits to Cindy Shin ( Owner of Template)
   Detail and Descriptive README.me file
   Added Live Server Demo: [Try It!](https://ramiaquista.github.io/conference-website/)
   Added Informative video about my job:  [Project Video](https://www.loom.com/share/a4161791d7764bf29385445001a65e58)

### CODE REVIEW

File: about-page.html [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+            2015 Electronic Entertainment Expo. Some rights reserved.<br />License
+            and Credits to Cindy Shin:
+            <a
+              href="https://www.behance.net/adagio07"
+              >https://www.behance.net/adagio07</a
+            >
+          </p>
+        </div>
+      </div>
+    </footer>
```
Comment (emmanuellekamwa) [inline about-page.html:227]: - [x] Hi. Good job on the footer but there is a little thing that is missing. It is a requirement that on mobile view the background color of the footers for the 2 pages become white but on the mobile view of the home page, the footer is not even present. Please kindly fix that.

File: about-page.html [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+            <ul id="options-list">
+              <li><a href="index.html">Home</a></li>
+              <li><a href="about-page.html">About</a></li>
+              <li><a href="index.html#program">Program</a></li>
+              <li><a href="index.html#program">Join</a></li>
+              <li><a href="index.html#sponsor">Sponsor</a></li>
+              <li><a href="#past-events">Events</a></li>
+            </ul>
+          </div>
+        </section>
```
Comment (emmanuellekamwa) [inline about-page.html:43]: - [x] I think that it would have been better here if you had used a nav tag instead of a section tag.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛑🛠🛑**
Hello @ramiaquista 👋🏽
Good job so far! 👌🏽💪🏽
I really like the way your project is presented, especially the UI and the code is well structured but there are some minor issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about-page.html (hunk 1 above)
Comment (ramiaquista) [reply about-page.html:227]: Thank's for the advice, I will improve my code. 😄

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @ramiaquista, 

Great work on making changes requested by the previous reviewer. 
You are using semantic `<nav>` tag for both of your menus now :clap:  
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**